docs/user-installation-review.md

```diff
@@ -0,0 +1,180 @@
+
+This is the utility to help you to troubleshoot the slow requests into your cluster, and help to take some decisions like changing the flavor of the block device used by the control plane, increasing IOPS, changing the flavor of the instances, etc.
+
+There's no magic or desired number, but for reference, based on the observations from integrated platforms, is to have no more than 30-40% of requests above 500ms while running the certification tests.
+
+> TODO: provide guidance on how to get the errors from the etcd pods, and parse it into buckets of latency to understand the performance of the etcd while running the certification environment.
+
+- Export the location you must-gather has been extracted:
+
+```bash
+export MUST_GATHER_PATH=${PWD}/must-gather.local.2905984348081335046
+```
+
+- Extract the utility from the tools repository:
+
+```bash
+TOOLS_IMAGE=$(skopeo inspect docker://quay.io/ocp-cert/tools:latest | jq .Digest)
+oc image extract ${TOOLS_IMAGE} --file="/usr/bin/insights-ocp-etcd-logs"
+chmod u+x insights-ocp-etcd-logs
+```
+
+- Generate the overall report
+
+> Note: This report can be useless depending on the history of logs. We recommend looking the next report which aggregate by the hour, so you can check the time frame the certification has been executed
+
+> TODO: the tool `insights-ocp-etcd-logs` could read all the logs from stdin, filter by str, then report the buckets - avoiding extracting complex greps
+
+```bash
+grep -rni "apply request took too long" ${MUST_GATHER_PATH} \
+    | grep -Po 'took":"([a-z0-9\.]+)"' \
+    | awk -F'took":' '{print$2}' \
+    | tr -d '"' \
+    | ./insights-ocp-etcd-logs
+```
+
+- Generate the overall report aggregated by the hour:
+
+> TODO: also improve the tool `insights-ocp-etcd-logs` to report by hour - maybe using flags
+
+```bash
+FILTER_MSG="apply request took too long"
+for TS in $( grep -rni "${FILTER_MSG}" ${MUST_GATHER_PATH} \
+    | awk '{print$1}' \
+    | awk -F'.log:' '{print$2}' \
+    | awk -F':' '{print$2}' \
+    | sort | uniq); do
+    echo "-> ${TS}"
+    grep -rni "${FILTER_MSG}" ${MUST_GATHER_PATH} \
+        | grep $TS \
+        | grep -Po 'took":"([a-z0-9\.]+)"' \
+        | awk -F'took":' '{print$2}' \
+        | tr -d '"' \
+        | ./insights-ocp-etcd-logs
+done
+```
+
+#### Mount /var/lib/etcd in sepparate disk
+
+One way to improve the performance on etcd is to use a dedicated block device.
+
+You can mount `/var/lib/etcd` by following the documentation:
+
+- [OpenShift Docs: Disk partitioning](https://docs.openshift.com/container-platform/4.11/installing/installing_bare_metal/installing-bare-metal.html#installation-user-infra-machines-advanced_disk_installing-bare-metal)
+- [KCS: Mounting separate disk for OpenShift 4 etcd](https://access.redhat.com/solutions/5840061)
+
+### Image Registry
+
+- persistent storage defined for image-registry
```

Comment:
I think we should link to the platform agnostic guide here: https://docs.openshift.com/container-platform/4.11/installing/installing_platform_agnostic/installing-platform-agnostic.html#installation-registry-storage-config_installing-platform-agnostic

Reply:
Mentioned it alongside other links I find in our doc. Thanks